frontends/election-manager/src/components/smartcard_modal/status_message.tsx

```diff
@@ -74,34 +78,36 @@ export function StatusMessage({
   }
 
   if (action === 'Program' && programmedUser) {
+    // No need to include the programmed user role in these messages since it's so prominent
+    // elsewhere on the screen
     return (
-      <Text success>
-        New {actionRoleReadableString} card has been programmed.
-        {'passcode' in programmedUser && (
+      <Text as="span" success>
+        {'passcode' in programmedUser ? (
           <React.Fragment>
-            <br />
-            The card PIN is {hyphenatePin(programmedUser.passcode)}. Write this
-            PIN down.
+            New card PIN is{' '}
+            <strong>{hyphenatePin(programmedUser.passcode)}</strong>.
           </React.Fragment>
+        ) : (
+          <React.Fragment>New card created.</React.Fragment>
         )}
       </Text>
     );
   }
 
   if (action === 'PinReset' && programmedUser && 'passcode' in programmedUser) {
+    // No need to include the programmed user role in these messages since it's so prominent
+    // elsewhere on the screen
```

Comment:
Is this comment necessary as it mentions something that is no longer on the screen?

Reply:
Covered in discussion on https://github.com/votingworks/vxsuite/pull/2267#discussion_r937741413